test/test_utils.py

```diff
@@ -79,6 +79,15 @@ def test_save_image_single_pixel_file_object(self):
             self.assertTrue(torch.equal(F.to_tensor(img_orig), F.to_tensor(img_bytes)),
                             'Pixel Image not stored in file object')
 
+    def test_draw_boxes(self):
+        img = torch.randint(0, 255, (3, 226, 226), dtype=torch.uint8)
+        boxes = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                             [10, 15, 30, 35], [23, 35, 93, 95]], dtype=torch.float)
+        labels = ['a', 'b', 'c', 'd']
+        utils.draw_bounding_boxes(img, boxes, labels=labels)
```

Comment:
can you also add a test checking that the color of the output image at pixel values `out[:, 0, 0:100] == fillcolor` etc, so that we know that we are masking the correct pixels in the image?

Reply:
I agree we should test pixels but I would rather test all functionalities including labels, fonts etc. I wonder if that's possible or if it will crate a flaky test due to differences on fonts across platforms. I'll give a try to test what I proposed on the earlier comment and see if this works.

Reply:
See latest code for the proposed approach of testing.